test/powershell/Language/Parser/Parsing.Tests.ps1

```diff
@@ -459,3 +459,20 @@ Describe "Ternary Operator parsing" -Tags CI {
         $expr.IfFalse | Should -BeOfType System.Management.Automation.Language.ConstantExpressionAst
     }
 }
+
+Describe "ParserError type tests" -Tag CI {
+    # This test was added because there use to be a hardcoded newline in the ToString() method of
+    # the ParseError class. This makes sure the proper newlines are used.
+    It "Should use consistant newline depending on OS" {
+        $expected = @'
+At line:1 char:5
++ $x =
++     ~
+You must provide a value expression following the '=' operator.
```

Comment:
Localized message is not good in test. We could remove it and check that right newline is after the extend.

Reply:
I've reworked the test.